src/Parser/CameraCalibrationParser.js

```diff
@@ -0,0 +1,80 @@
+import * as THREE from 'three';
+import OrientedImageCamera from '../Renderer/OrientedImageCamera';
+
+/**
+ * The CameraCalibrationParser module provide a [parse]{@link module:CameraCalibrationParser.parse}
+ * method that takes a JSON array of camera calibrations in and yields an array of THREE.Camera
+ *
+ * @module CameraCalibrationParser
+ */
+
+THREE.Matrix4.prototype.setMatrix3 = function setMatrix3(m) {
+    this.elements[0] = m.elements[0];
+    this.elements[1] = m.elements[1];
+    this.elements[2] = m.elements[2];
+    this.elements[4] = m.elements[3];
+    this.elements[5] = m.elements[4];
+    this.elements[6] = m.elements[5];
+    this.elements[8] = m.elements[6];
+    this.elements[9] = m.elements[7];
+    this.elements[10] = m.elements[8];
+    return this;
+};
+
+// the json format encodes the following transformation:
```

Comment:
This description should be below in the `parse` documentation, so we can read it in the generated documentation, or above in the `module` section.